dhis-2/dhis-support/dhis-support-system/src/main/java/org/hisp/dhis/system/database/HibernateDatabaseInfoProvider.java

```diff
@@ -207,6 +237,62 @@ private boolean isSpatialSupport()
         }
     }
 
+    /**
+     * Attempts to create a pg_trgm database extension. Checks if extension is
+     * created
+     */
+    private boolean isTrigramExtensionCreated()
+    {
+        try
+        {
+            jdbcTemplate.execute( "create extension pg_trgm;" );
+        }
+        catch ( Exception ex )
+        {
```

Comment:
could you please add a comment inside of this and subsequent catch clauses, so we know when reading this why we catch all exceptions without handling them.

Reply:
Noted! Will add